index.html

```diff
@@ -106,6 +106,60 @@ <h3>
       <h2>
         Examples
       </h2>
+      <h3>
+        Loading and parsing
+      </h3>
+      <p>
+        A model element whose source path resolves to a valid resource
+        dispatch a `load` event upon the successful fetching and 
+        presentation of the resource data. If the asset fails to resolve,
+        for instance due to an invalid URL, a failure to retrieve the bytes,
+        or a failure in the parsing process, the element will dispatch an 
+        `error` event.
+      </p>
```

Comment:
We probably want and actual example here:

```suggestion
      </p>
      <div class="example" title="Error handling">
      <pre class="js">
      const model = document.querySelector("model");
      model.addEventListener("error", () => {
        // Handle error 
      });
      </pre>
```